### PR TITLE
Chores before publishing

### LINKED ISSUE #21: README

### LINKED ISSUE #22: User guide

### LINKED ISSUE #23: Documentation

### LINKED ISSUE #29: Deployment

### PR DESCRIPTION
**Pull request by 0xGabi**
I think this closes #29, closes #23, closes #22, closes #21 

Task done:

- [x] Rename to `Time Lock` 

- [x] Clean up before publish

- [x] Include metadata (description, images and icon)

TODO:

- [x] Fix tests

- [x] Update front-end with Time Lock rename

### CODE REVIEW

**Comment by 0xGabi** [comment]
@burrrata I have included your documentation from #42 

What do you think of close #21 and have a new issue only for Rinkeby deployment?

File: README.md [hunk 1]
```diff
@@ -1,57 +1,61 @@
...
 
 The code in this repo has not been audited.
 
 ## How does it work
 
-The lock app is a [forwarder](https://hack.aragon.org/docs/forwarding-intro). By granting the lock app a permission like `Create Votes` the user will be prompted and required to lock tokens before the user's intent can be forwarded.
+The Time Lock app is a [forwarder](https://hack.aragon.org/docs/forwarding-intro). By granting the Time Lock app a permission like `Create Votes` the user will be prompted and required to lock tokens before the user's intent can be forwarded.
 
-We do not protect the lock function with a role, so anyone is able to make locks and forward actions. We keep track of when deposits are made and by whom so that users are only able to re-claim deposits that they have made after the duration has elapsed.
+We do not protect the lock function with a role, so anyone is able to make locks and forward actions. We keep track of when locks are made and by whom so that users are only able to re-claim locks that they have made after the duration has elapsed.
```
Comment (fabriziovigevani) [inline README.md]: This should be removed as we now check with an Oracle if someone can lock tokens:  `We do not protect the lock function with a role, so anyone is able to make locks and forward actions.`

File: README.md [hunk 2]
```diff
@@ -1,57 +1,61 @@
...
 
 ### Initialization
 
-The deposits app is initialized with a `token`, `_lockDuration`, and `_lockAmount` parameters which determines how long deposits are locked and the amount to deposit.
-The `token` parameter can be set at initialization and not changed. If a change is necessary the user can install a new instance and change permissions in the organization to reflect the change.
+The Time Lock app is initialized with a `_token`, `_lockDuration`, and `_lockAmount` parameters which determines the type of token that can be locked, how long locks are locked the amount of tokens to lock.
```
Comment (fabriziovigevani) [inline README.md]:
```suggestion
The Time Lock app is initialized with a `_token`, `_lockDuration`, `_lockAmount` and `_spamPenaltyFactor` parameters which determines the token to be locked, how long tokens are locked, the amount of tokens to lock and a penalty percentage for spamming proposals.
```

File: README.md [hunk 3]
```diff
@@ -81,8 +85,8 @@ You will also see the configuration for your local deployment in the terminal. I
 
 ### Template
 
-The Lock app is initialized with a `_lockDuration` of 1:30 minutes and a `_lockAmount` of 20 LKT tokens.
-The app has the permission to create votes, so if you try to mint yourself some tokens from the `tokens` app it will first prompt you to approve the Lock app to transfer 20 LKT tokens to the contract on your behalf.
+The Time Lock app is initialized with a `_lockDuration` of 1:30 minutes and a `_lockAmount` of 20 LKT tokens.
```
Comment (fabriziovigevani) [inline README.md]:
```suggestion
The Time Lock app is initialized with a `_lockDuration` of 1:30 minutes, a `_lockAmount` of 20 LKT tokens and a `_spamPenaltyFactor` of 100%.
```

**Comment by 0xGabi** [comment]
@fabriziovigevani you think we need to made some update to front-end with respect with the Time Lock rename?

**Comment by fabriziovigevani** [comment]
Probably Only the Title name.

Suggested some changes in the README to add the `spamPenaltyFactor`. 

Noticed also that the technical documentation has to be updated also but we can submit an issue and refactor it later.

**Pull request merged**